week2/2-website/css/style.css

```diff
@@ -1,19 +1,389 @@
+ *
+ * Features
+ *
  */
 
+.features {
+    min-width: 600px;
+    padding: 0 .938rem;
+    margin-bottom: 6.4rem;
+}
+
+.features__heading {
+    color: var(--text-color);
+    font-weight: 300;
+    font-size: 2.75rem;
+    text-align: center;
+    line-height: 2.8rem;
+    margin-top: 8rem;
+    margin-bottom: 3.2rem;
+}
+
+.features__list {
+    display: flex;
+    margin: 0 auto;
+    padding: 0;
+    max-width: 700px;
+}
+
+.features__list>li {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    align-items: center;
+}
+
+.features__image {
+    width: 127px;
+    height: 127px;
+}
+
+.features__description {
+    font-size: 1.6rem;
+    line-height: 1.6rem;
+    color: var(--text-color);
+    text-align: center;
+    margin-top: 2.5rem;
+}
+
+/*=====  End of MAIN  ======*/
+
+
+/*=============================================
+=            FOOTER            =
+=============================================*/
+
+.footer__container {
+    padding: 0 .938rem 8.5rem .938rem;
+    min-width: 600px;
+}
+
+.footer {
+    border-top: 1px solid var(--separator-color);
+    text-align: center;
+    margin: 0 auto;
+}
+
+.footer__infoText {
+    margin-top: 1.3rem;
+    margin-bottom: 1.25rem;
+}
+
+.footer__socialLinks {
+    margin-bottom: 1.3rem;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    list-style-type: none;
+}
+
+.footer__socialLinks__link {
+    text-decoration: none;
+    height: 2.5rem;
+    width: 2.5rem;
+    border: 1px solid var(--separator-color);
+    border-radius: 50px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.footer__socialLinks>li {
+    margin-right: 1rem;
+}
+
+.footer__socialLinks>li:last-of-type {
+    margin-right: 0;
+}
+
+.icon--twitter {
+    color: var(--twitter-color);
+}
+
+.icon--facebook {
+    color: var(--facebook-color);
+}
+
+.icon--instagram {
+    color: var(--instagram-color);
+}
+
+.footer_copyrightText {
+    font-size: .9rem;
+    font-weight: 300;
+    color: #838994;
+}
+
+/*=====  End of FOOTER  ======*/
+
+
+
+/*=============================================
+=            MEDIA QUERIES            =
+=============================================*/
+
+@media screen and (min-width: 768px) {
+
+    /**
+     *
+     * HEADER
+     *
+     */
+
+    .header {
+        max-width: 736px;
+    }
+
+
+    /**
+     *
+     * FOOTER
+     *
+     */
+
+    .footer {
+        max-width: 736px;
+    }
+}
+
+@media screen and (min-width: 992px) {
+
+    /**
+     *
+     * HEADER
+     *
+     */
+
+    .header {
+        max-width: 960px;
+    }
+
+
+    /**
+     *
+     * MAIN
+     *
+     */
+
+    .features__list {
+        max-width: 920px;
+    }
+
+    /**
+     *
+     * FOOTER
+     *
+     */
+
+    .footer {
+        max-width: 930px;
+    }
+
+}
+
+@media screen and (min-width: 1200px) {
+
+    /**
+     *
+     * HEADER
+     *
+     */
+
+    .header {
+        max-width: 60%;
```

Comment:
60% of 1200px is about 720px so the width of some of these items get's slightly smaller than the previous media query here.

You could perhaps have added a width of 60% and a min-width of 960px so it scales a little more cleanly.

Reply:
Yes, it is a bit strange here how it goes from wider header to a more narrow while increasing the width of a window but that's how it behaves in the original.